src/ape/managers/chain.py

```diff
@@ -146,8 +146,7 @@ def poll_blocks(
             start (Optional[int]): The block number to start with. Defaults to the pending
               block number.
             required_confirmations (Optional[int]): The amount of confirmations to wait
-              before yielding the block. The more confirmations, the less likely a reorg
-                will occur.
+              before yielding the block. The more confirmations, the less likely a reorg will occur.
```

Comment:
Just fixing the indent here because it was causing a docs warning.

No content changes!